ci-operator/step-registry/gather/ibmcloud-resource/gather-ibmcloud-resource-commands.sh

```diff
@@ -170,7 +201,13 @@ if [ -f ${SHARED_DIR}/metadata.json ]; then
     RESOURCE_GROUP=$(jq -r .ibmcloud.resourceGroupName ${SHARED_DIR}/metadata.json)
     echo "Resource group: $RESOURCE_GROUP"
 fi
-# Enable exit on error to short circuit if there are failures during gather
+
+if [ -f "${CLUSTER_PROFILE_DIR}/ibmcloud-cis" ]; then
+    ibmcloud_cis_instance_name="$(cat "${CLUSTER_PROFILE_DIR}/ibmcloud-cis")"
+else
+    ibmcloud_cis_instance_name="Openshift-IPI-CI-CIS"
+    BASE_DOMAIN="ci-ibmcloud.devcluster.openshift.com"
```

Comment:
You already introduced `BASE_DOMAIN` ENV var on the step ref yaml, why not set its default value to it ?